test/Libraries/GeometryColorTests/GeometryColorTests.csproj

```diff
@@ -9,19 +9,22 @@
     <RootNamespace>DisplayTests</RootNamespace>
     <AssemblyName>DisplayTests</AssemblyName>
   </PropertyGroup>
-  <ItemGroup>
-	  <PackageReference Include="NUnit" Version="2.6.3" />
-    <PackageReference Include="DynamoVisualProgramming.LibG_229_0_0" Version="2.18.0.1400"/>
-    <PackageReference Include="NUnitTestAdapter" Version="2.3.0" ExcludeAssets="all" />
-      <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.3.2" />
```

Comment:
Don't these references (LibG and Test.Sdk) need to be there for all targets? I see that they've been removed altogether, why?